### DISCUSSION TITLE
No SK data after connection to server?

### DISCUSSION THREAD

**Discussion by Ranur** [hatlabs/discussions#101] [Q&A]
![20240304_211743](https://github.com/hatlabs/discussions/assets/45295170/4adb53c3-ac09-4341-beb2-5949d83733ec)
I got the Hamlet set up, used the sample software, connected to my Rpi access point, accepted the connection to SignalK server...
But when looking at the data browser or the signalk dashboards, none of the sensors are visible. What have I missed?

**Comment by big-col** [comment]
What can you see in the terminal window?

**Reply by Ranur** [reply]
![20240310_200015](https://github.com/hatlabs/discussions/assets/45295170/ef6da224-424e-47dc-bfd7-e772691b4e41)
This for instance. The Halmet is added to the server, but if I go into the data browser, I can't se any of the signalk paths I have added to the Halmet. Note: I haven't physically added the sensors yet. Could that be it?

**Reply by big-col** [reply]
You need to add the sensors before any data, or the data paths, will appear.  You can also  create some test data in SK. There is a test plugin available to install that I've used a couple of times...

**Reply by Ranur** [reply]
Ok! So just any temp sensor or a tank sensor?

**Reply by big-col** [reply]
I think you can emulate anything, let me check and confirm

**Reply by Ranur** [reply]
Now I have added a tank meter, but I still see nothing. No data or no site of the Halmet in the Dashboard and "Connections Activity,"

**Comment by allancph** [comment]
I have exactly the same problem. I am simulating RPM and tank level, and it's showing fine on my plotter n2k. Halmet made an access request to SK, but deliver no data.

**Comment by allancph** [comment]
So finally got this working. 

#define ENABLE_SIGNALK // missing everywhere but main.cpp

to beginning of all dot cpp files

erase HALMET, build, upload, configure

done..
